charts/streams-app/values.yaml

```diff
@@ -50,6 +50,21 @@ jmx:
   metricRules:
     - ".*"
 
+autoscaling:
+  enabled: false
+  # consumergroup: foo
+  lagThreshold: "1000"
```

Comment:
How about this value? Is it ok to set a default here?

Reply:
I think it is not that critical but should also be configured for every deployment, right?

Reply:
Yes, it might be different for every deployment